servers/src/main/java/tachyon/web/WebInterfaceWorkerBlockInfoServlet.java

```diff
@@ -201,19 +201,17 @@ private UiFileInfo getUiFileInfo(TachyonFS tachyonClient, int fileId, TachyonURI
 
     UiFileInfo uiFileInfo = new UiFileInfo(fileInfo);
     boolean blockExistOnWorker = false;
-//    for (long blockId : fileInfo.getBlockIds()) {
-//      for (StorageDir storageDir : mWorkerStorage.getStorageDirs()) {
-//        if (storageDir.containsBlock(blockId)) {
-//          blockExistOnWorker = true;
-//          long blockSize = storageDir.getBlockSize(blockId);
-//          long blockLastAccessTime = storageDir.getLastBlockAccessTimeMs(blockId);
-//          StorageLevelAlias storageLevelAlias =
-//              StorageDirId.getStorageLevelAlias(storageDir.getStorageDirId());
-//          uiFileInfo.addBlock(storageLevelAlias, blockId, blockSize, blockLastAccessTime);
-//          break;
-//        }
-//      }
-//    }
+    for (long blockId : fileInfo.getBlockIds()) {
+      if (mBlockDataManager.hasBlockMeta(blockId)) {
+        blockExistOnWorker = true;
+        BlockMeta blockMeta = mBlockDataManager.getBlockMeta(blockId);
+        long blockSize = blockMeta.getBlockSize();
+        StorageLevelAlias storageLevelAlias =
+            StorageDirId.getStorageLevelAlias(blockMeta.getParentDir().getStorageDirId());
+        // The block last access time is not available. Use -1 for now.
+        uiFileInfo.addBlock(storageLevelAlias, blockId, blockSize, -1);
+      }
+    }
```

Comment:
I think this can be simplified by just directly trying `getBlockMeta`. I don't think `FileDoesNotExistException` is required to be thrown and can just be the `IOException` thrown by `getBlockMeta` for now.